twitter/twitter.go

```diff
@@ -0,0 +1,43 @@
+package twitter
+
+import (
+	"fmt"
+
+	"github.com/dghubble/go-twitter/twitter"
+	"github.com/dghubble/oauth1"
+)
+
+// Client ...
+type Client struct {
+	ConsumerKey    string
```

Comment:
These fields do not need to be exported.

Reply:
Fixed